### PR TITLE
docs: Upload Voice Verification SRS Document (Ver 1.0)

### PR DESCRIPTION
**Pull request by hantt-8**
# Description
Change SRS document (docx) to .md file and upload to company documentation repo
## Type of change
- [x] Documentation (update or new)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have requested a review from ... on the Pull Request

### CODE REVIEW

**Comment by hantt-8** [comment]
@ShaeChristmas @DJ2507 Can any one of you please review to this docs if there's anything I need to update or it is okay to go through? (I'm not sure if doing this is requiring a review :')

**Comment by hantt-8** [comment]
@ShaeChristmas @DJ2507 Can any one of you please review to this docs if there's anything I need to update or it is okay to go through? (I'm not sure if doing this is requiring a review :')

**Review by ShaeChristmas — APPROVED** [review]
Document appears all good to me; also passes all checks.

File: docs/OnTrack/Voice Verification/Voice Verification SRS Document.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+# Product Purpose
+
+The Purpose of the Voice Verification System for OnTrack is to add the ability for Voice Samples
+that are submitted to OnTrack to undergo a verification process to ensure that the speaker in the
+sample is the same as the person taking part in the unit. Specifically, this is to identify when a
+student is contract cheating, or if the person in the specific submission is also the person
+undertaking the unit.
+
+The Intended audience for this project is the users of OnTrack; both students for using the system
```
Comment (ginge270) [inline docs/OnTrack/Voice Verification/Voice Verification SRS Document.md:13]: Lower case i for "Intended"

Comment (ginge270) [reply docs/OnTrack/Voice Verification/Voice Verification SRS Document.md:13]: And also p for "Purpose" on line 7

Comment (ginge270) [reply docs/OnTrack/Voice Verification/Voice Verification SRS Document.md:13]: s for "Scope" line 22

File: docs/OnTrack/Voice Verification/Voice Verification SRS Document.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+- Students do submit an enrolment voice file for later comparison.
+- Tutors use the system whenever a voice submission is required.
+- The same person is speaking throughout the entirety of the voice files.
+
+Each of these assumptions is important for the use and requirements of the system. The system should
+be able to deal with multiple requests in quick succession, be actively deployed to the OnTrack
+System, and have strict requirements for the initial voice file.
+
+Further, a few different aspects are relied upon for the project to function. These include:
```
Comment (ginge270) [inline docs/OnTrack/Voice Verification/Voice Verification SRS Document.md:57]: Furthermore?

File: docs/OnTrack/Voice Verification/Voice Verification SRS Document.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+Additionally, for OnTrack to function correctly, the full system (Frontend and API) needs to be
+deployed and using the Python Container effectively.
+
+# System Requirements
+
+## Functional Requirements
+
+The functional requirements of the system are as follows:
+
+- The System should be able to accept an Enrolment voice file for later comparison.
```
Comment (ginge270) [inline docs/OnTrack/Voice Verification/Voice Verification SRS Document.md:73]: System and enrolment are both capitalised

**Review by ginge270 — APPROVED** [review]
Have left a few comments about a few small grammar issues but other than that awesome job!!

**Pull request merged**